Gemfile.lock

```diff
@@ -1,39 +1,60 @@
 PATH
   remote: .
   specs:
-    deadlock_retry (1.2.0)
+    deadlock_retry (2.2.1)
 
 GEM
   remote: http://rubygems.org/
   specs:
-    activemodel (6.1.7.2)
-      activesupport (= 6.1.7.2)
-    activerecord (6.1.7.2)
-      activemodel (= 6.1.7.2)
-      activesupport (= 6.1.7.2)
-    activesupport (6.1.7.2)
+    activemodel (7.1.5.1)
+      activesupport (= 7.1.5.1)
+    activerecord (7.1.5.1)
+      activemodel (= 7.1.5.1)
+      activesupport (= 7.1.5.1)
+      timeout (>= 0.4.0)
+    activesupport (7.1.5.1)
+      base64
+      benchmark (>= 0.3)
+      bigdecimal
       concurrent-ruby (~> 1.0, >= 1.0.2)
+      connection_pool (>= 2.2.5)
+      drb
       i18n (>= 1.6, < 2)
+      logger (>= 1.4.2)
       minitest (>= 5.1)
+      mutex_m
+      securerandom (>= 0.3)
       tzinfo (~> 2.0)
-      zeitwerk (~> 2.3)
-    concurrent-ruby (1.2.0)
-    i18n (1.12.0)
+    base64 (0.2.0)
+    benchmark (0.4.0)
+    bigdecimal (3.1.9)
+    byebug (11.1.3)
+    concurrent-ruby (1.3.5)
+    connection_pool (2.5.0)
+    drb (2.2.1)
+    i18n (1.14.7)
       concurrent-ruby (~> 1.0)
-    minitest (5.17.0)
+    logger (1.6.6)
+    minitest (5.25.5)
+    minitest-color (0.0.2)
+      minitest (~> 5)
     mocha (2.0.2)
       ruby2_keywords (>= 0.0.5)
+    mutex_m (0.3.0)
     ruby2_keywords (0.0.5)
+    securerandom (0.4.1)
+    timeout (0.4.3)
     tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
-    zeitwerk (2.6.6)
 
 PLATFORMS
   aarch64-linux
 
 DEPENDENCIES
-  activerecord (~> 6.0)
+  activerecord (~> 7.1.5.1)
+  byebug
```

Comment:
The only reason we use byebug at all is because it works better with our selenium testing.  If I could figure out how to make the standard debugger work well, we'd ditch it.